src/SignalR/server/Core/src/Internal/TypedClientBuilder.cs

```diff
@@ -64,14 +67,19 @@ private static Type GenerateInterfaceImplementation(ModuleBuilder moduleBuilder)
             BuildMethod(type, method, proxyField);
         }
 
-        return type.CreateTypeInfo()!;
+        return type.CreateType();
     }
 
-    private static IEnumerable<MethodInfo> GetAllInterfaceMethods(Type interfaceType)
+    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2062:UnrecognizedReflectionPattern",
```

Comment:
Note the suppressions in this file are the same as in https://github.com/dotnet/runtime/blob/3ec76324514868b0ee4cfcca43ef96a4d62d44fa/src/libraries/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs#L168-L174